lib/membrane_rtc_engine/track.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Membrane.RTC.Engine.Track do
+  @moduledoc """
+  Module representing a WebRTC track.
+  """
+  alias ExSDP.Attribute.FMTP
+
+  @enforce_keys [:type, :stream_id, :id, :fmtp]
+  defstruct @enforce_keys ++ [encoding: nil, format: nil, disabled?: false]
+
+  @type id :: String.t()
+  @type encoding :: :OPUS | :H264 | :VP8
+  @type format :: :RTP | :raw | :RTMP
+
+  @type t :: %__MODULE__{
```

Comment:
I would add some docs for this type. Take a look at https://elixirforum.com/t/how-to-properly-document-structs-for-exdoc/8892/2